util/aggregation_util.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions for aggregations on dictionaries and strings."""
+
+from absl import logging
+
+
+def aggregate_value(value1: str, value2: str, aggregate: str = 'sum') -> str:
```

Comment:
1. from the comments, maybe make it src, target?
2. should the type be `Any` instead of str perhaps?

Reply:
1. Changed args to src, dst

2. set arg type to Union[str, int, float, list, set]